e2e/suite/job_generate_test.go

```diff
@@ -67,6 +68,10 @@ var _ = Describe("Job Generation", func() {
 			Expect(jobs[0].Status.Succeeded).To(BeNumerically("==", 1))
 			Expect(jobs[0].Status.Active).To(BeNumerically("==", 0))
 			Expect(jobs[0].Status.Failed).To(BeNumerically("==", 0))
+
+			plan, err = e2e.GetPlan(plan.Name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(upgradeapiv1.PlanComplete.IsTrue(plan)).To(BeTrue())
```

Comment:
You might need to wrap this with an Eventually?

https://onsi.github.io/ginkgo/#using-speccontext-with-gomegas-eventually

Reply:
Fixed, thanks.